.ci/retry.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
+set -euo pipefail
 
-numRetries=3
-retryDelay=1
+readonly numRetries=3
```

Comment:
Bash coding conventions usually use lower snake case. But that is not so important.
https://github.com/easybash/bash-coding-style-guide#variable

Reply:
I will keep this in mind for the future.